src/management-system-v2/components/processes/index.tsx

```diff
@@ -170,7 +170,26 @@ const Processes = ({
     { dependencies: [selectedRowKeys.length] },
   );
 
-  const createProcessButton = <ProcessCreationButton wrapperElement="Create Process" />;
+  const createProcessButton = (
+    <ProcessCreationButton
+      wrapperElement="Create Process"
+      defaultOpen={
+        typeof window !== 'undefined' &&
+        new URLSearchParams(document.location.search).has('createprocess')
+      }
+    />
+  );
+
+  useEffect(() => {
+    const searchParams = new URLSearchParams(document.location.search);
+    if (searchParams.has('createprocess')) {
+      searchParams.delete('createprocess');
+      router.replace(
+        window.location.origin + window.location.pathname + '?' + searchParams.toString(),
+      );
+    }
+  }, []);
```

Comment:
Would it make sense to do this when the modal is closed? If you reload the page (or it loads on the next day) the modal would be gone, even if the user never saw it.

Reply:
Yeah, that makes sense, to do it I had to refactor processCreationButton and processModal